src/Microsoft.IIS.WebManager/app/certificates/certificates-list.component.ts

```diff
@@ -121,6 +120,10 @@ export class CertificatesListComponent implements OnInit, OnDestroy {
         }
     }
 
+    get serviceLoading() {
```

Comment:
If we're adding back privates because we aren't using AOT, then we don't need these getters which are just fluff at this point.

Reply:
I didn't want to remove this. Eventually we would move to AOT. It's not hurting anything.